app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```
@@ -38,6 +38,9 @@ def refresh
             post_refresh_ems_cleanup(ems, targets)
           end
 
+          # We need to reload the ems, since it can try to autosave a models that did not pass a validation, and the
+          # ems status with the error would not save.
+          ems.reload
```

exactly.
and right now it just silently fails